onnxruntime/python/tools/quantization/operators/qdq_base_operator.py

```diff
@@ -19,4 +19,7 @@ def quantize(self):
             nodes_to_iterate = itertools.chain(node.input, node.output)
 
         for tensor_name in nodes_to_iterate:
-            self.quantizer.quantize_tensor(tensor_name)
+            if self.quantizer.is_per_channel():
+                self.quantizer.quantize_tensor_per_channel(tensor_name, self.quantizer.qdq_channel_axis)
```

Comment:
what if the operator doesn't support per-channel

Reply:
cpu ep only per-channel for Conv and Matmul now

Reply:
Add a new extra options OpTypesSupportPerChannelQuantization